src/OmniSharp.Host/Services/AssemblyLoader.cs

```diff
@@ -77,5 +81,31 @@ public Assembly LoadFrom(string assemblyPath)
             _logger.LogTrace($"Assembly loaded from path: {assemblyPath}");
             return assembly;
         }
+
+        public Assembly LoadFrom(string assemblyPath, bool lockAssembly)
```

Comment:
Should the two `LoadFrom` overloads be merged with `lockAssembly` becoming an optional parameter? Also, from an API perspective, I'd sort of prefer to see `dontLockAssemblyOnDisk` since the goal in calling this method is to avoid locking.

Reply:
I could merge if you like. And yeah, the naming `dontLockAssemblyOnDisk` sounds better, with a default to `false`.